jsettlers.common/src/main/resources/jsettlers/common/buildings/big_temple.xml

```diff
@@ -148,9 +148,8 @@
 
 	<door dx="0" dy="2" />
 
-	<constructionStack dx="5" dy="6" material="STONE" buildrequired="8" />
+	<constructionStack dx="5" dy="6" material="STONE" buildrequired="14" />
```

Comment:
Can't we leave this to be two stacks?

This is something that annoyed me in the original game - you always need to wait until the rest of the materials are brought to the construction site.